index.html

```diff
@@ -20,7 +19,7 @@
 
     <body>
         <div id="app"></div>
-        <script>Raven.config('https://23e9c0c37a3841b8a692c98b8978f515@sentry.io/147247').install();</script>
         <script src="scripts/src/app.js"></script>
+        <script>Raven.config('https://23e9c0c37a3841b8a692c98b8978f515@sentry.io/147247').install();</script>
```

Comment:
@rickr sorry for late responses, the og:image property is what shows up when you share the website on social media sites like FB. It's the default image that their API grabs. not sure if it's just FB that uses og:image. I think twitter uses twitter:share or twitter:image. This is off the top of my head, I suppose I could give you a better answer, but you get the idea.

The script rearrangement - I was just unaware of their special documentation. Generally all of your js goes towards the bottom, and it seemed like it was slowing down page load being at a higher order. I don't think it would hurt anything by being lower, but maybe I also don't fully understand the scope of what you're tracking, so it's hard to say. I don't think there's much you can track in that short window of ms with analytics though. I'll take a look at the docs to see if I can understand why they want it loaded before app code.